src/server/req_stat.c

```diff
@@ -748,6 +749,45 @@ req_stat_svr(struct batch_request *preq)
 		(void)reply_send(preq);
 }
 
+/**
+ * @brief
+ * 		status_sched - Build the status reply for single scheduler
+ *
+ * @param[in]	psched	-	ptr to sched receiving status query
+ * @param[in]	preq	-	ptr to the decoded request
+ * @param[in,out]	pstathd	-	head of list to append status to
```

Comment:
Do you use pstathd as input anywhere or do you just append to it?  If you just append to it and don't use it as input, then I'd call it an out.